includes/Core/Authentication/Authentication.php

```diff
@@ -1359,4 +1360,27 @@ private function filter_features_via_proxy( $feature_enabled, $feature_name ) {
 		return $feature_enabled;
 	}
 
+	/**
+	 * Invalid nonce error handler.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param string $action Action name.
+	 */
+	public static function invalid_nonce_error( $action ) {
+		if ( strpos( $action, 'googlesitekit_proxy_' ) !== 0 ) {
+			wp_nonce_ays( $action );
+			return;
+		} else {
+			// Copied from wp_nonce_ays() with tweak to the url.
+			$html  = __( 'The link you followed has expired.', 'default' );
```

Comment:
Even though the translation is available in the default text domain, we need to use our own text domain here to make sure we have translations even when this string is changed in the core.

```suggestion
			$html  = __( 'The link you followed has expired.', 'google-site-kit' );
```

Reply:
https://github.com/google/site-kit-wp/pull/3976/commits/72855cff21aa68eb4977874409f63f5ca88292a4